Gopkg.toml

```diff
@@ -27,7 +27,13 @@
 required = [
   "github.com/go-bindata/go-bindata/go-bindata",
   # Apply workaround from https://github.com/golang/dep/issues/1799
-  "gopkg.in/fsnotify.v1"
+  "gopkg.in/fsnotify.v1",
+  "k8s.io/client-go/plugin/pkg/client/auth/azure",
```

Comment:
This is redundant with the `import` in engine.go, but it seems more appropriate here.